docs/tutorials/RemovingFields.md

```diff
@@ -103,7 +103,7 @@ In `src/test/data/`, data meant for testing purposes are stored. While keeping t
   "persons": [ {
     "name": "Person with invalid name field: Ha!ns Mu@ster",
     "rating": "9482424",
-    "email": "hans@example.com",
+    "operatingHours": "hans@example.com",
     "address": "4th street"
```

Comment:
Should the data here be updated to match our new data? e.g. Rating is 1-5, operatingHours is a time, etc

Reply:
Maybe the changes here can be made in another issue or pull request, where we change outdated inputs accordingly?